source/administration/version-archive.rst

```diff
@@ -1,27 +1,35 @@
 Version Archive
 ===========================
 
+To verify the GPG signature of a Mattermost release, use the public key stored in Keybase: https://keybase.io/mattermost.
```

Comment:
Do customers understand why they would verify the gpg signature?  Would it be helpful to provide a little more information on how this is beneficial to them? 0/5-just a thought as I read through this.

Reply:
@wiersgallak Good suggestion. How would this sound?

```
If you want to check that the version of Mattermost you are installing is the official, unmodified version, compare the SHA-265 checksum or the file's GPG signature with the one published in this version archive. To verify the GPG signature of a Mattermost release, use the public key stored in Keybase: https://keybase.io/mattermost.
```

Reply:
Looks great!